packages/plans-grid-next/src/hooks/data-store/use-grid-plans.tsx

```diff
@@ -63,6 +64,13 @@ const isGridPlanVisible = ( {
 } ): boolean => {
 	let isVisible = planSlugsForIntent.includes( planSlug );
 
+	// TODO: actually check if the site is a big sky trial
+	const isBigSkyTrial = true;
```

Comment:
@jboland88 @p-jackson, Looking for some help here. I stubbed this out for now, as I'm not familiar enough with this code - what the best approach is for globally hiding customizing the plans shown on every plan grid across wordpress.com?